### DISCUSSION TITLE
is there a Plan to introduce Java SDk for Deepgram

### DISCUSSION THREAD

**Discussion by shehrooz-sattar** [deepgram/community#675] [General help] [answered]
is there a Plan to introduce Java SDK for Deepgram?

if there is, then where i can find the timeline for this feature and further details?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
We have been discussing this internally. Stay tuned!
